lib/hz2600/Kazoo/Api/Entity/Search.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kazoo\Api\Entity;
+
+/*
+TODO: Add support for accounts/{account_id}/search
+*/
+
+class Search extends AbstractEntity
+{
+    public function search($type, $value, $view)
+    {
+	$type		= urlencode($type);
+	$value		= urlencode($value);
+	$view		= urlencode($view);
+	$response	= $this->get(array(), "/search?t=$type&q=$view&v=$value");
+	
+	return $response->getData();
+    }
+    
+    public function searchAccounts($value, $view)
+    {
+	    return $this -> search("account", $value, $view);
+    }
+    
+    public function getAccountByRealm($realm)
+    {
+	    $result = $this -> searchAccounts($realm, "realm");
+	    
+	    foreach ($result as $account)
+	    {
+		    if ($account -> realm == $realm)
```

Comment:
Please remove spaces between object and method EG: $object->method to comply with style of this project.

Reply:
Done